### PR TITLE
Enable Subscription config in OperandRegistry

### PR DESCRIPTION
**Pull request by horis233**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: bundle/manifests/operator.ibm.com_operandconfigs.yaml [hunk 1]
```diff
@@ -62,7 +62,6 @@ spec:
                     spec:
                       additionalProperties:
                         type: object
-                        x-kubernetes-preserve-unknown-fields: true
```
Comment (Daniel-Fan) [inline bundle/manifests/operator.ibm.com_operandconfigs.yaml:65]: Should we keep this column?

File: config/crd/bases/operator.ibm.com_operandconfigs.yaml [hunk 2]
```diff
@@ -56,7 +60,6 @@ spec:
                     spec:
                       additionalProperties:
                         type: object
-                        x-kubernetes-preserve-unknown-fields: true
```
Comment (Daniel-Fan) [inline config/crd/bases/operator.ibm.com_operandconfigs.yaml:59]: And this column as well

Comment (horis233) [reply config/crd/bases/operator.ibm.com_operandconfigs.yaml:59]: nice catch!

Comment (horis233) [reply config/crd/bases/operator.ibm.com_operandconfigs.yaml]: Updated

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**